lib/widgets/payment_dialogs/processing_payment_dialog.dart

```diff
@@ -98,6 +103,10 @@ class ProcessingPaymentDialogState extends State<ProcessingPaymentDialog>
       if (widget.isLnurlPayment) {
         navigator.pop(err);
       }
+      if (err is FfiException) {
+        navigator.removeRoute(_currentRoute!);
```

Comment:
Wouldn't this cause a null exception if this error is received for `isLnurlPayment: true`?
We would be trying to remove a route that does not exist at this point.

This would need to be wrapped with: 
```
if(_currentRoute != null && _currentRoute!.isActive){
        navigator.removeRoute(_currentRoute!);
}
```

Reply:
Added thanks